target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -184,6 +184,10 @@ dlink,dap-1365-a1)
 dlink,dir-859-a1)
 	ucidef_set_led_switch "internet" "WAN" "green:internet" "switch0" "0x20"
 	;;
+dlink,dir-869-a1)
+	ucidef_set_led_switch "led_power_orange" "WAN" "orange:power" "switch0" "0x20"
+ 	ucidef_set_led_switch "led_power_white" "WAN" "white:power" "switch0" "0x20"
```

Comment:
Why bind both to the same trigger? Is this vendor behavior?

Reply:
Apart from that, if that's a power LED, it should not indicate ethernet at all.

Reply:
Ping.

Reply:
I don't know here, is there any list of all triggers?

Reply:
If you don't know what you did, let's start with you telling me what you wanted to do ...

Reply:
I was starting from commit # 3296, but I didn't find anything in this file that would be a suitable replacement for "WAN"

Reply:
I've always tried to inspire the dir-859, but there's only one line "green: internet", which is confusing to me because the dir-859 has more LEDs

Reply:
I built as it is here and after the boot, both LEDs blinking. I found that this file affects "/etc/config/system" and accordingly I tried to fix it so that the LEDs work as they should. I renamed it POWER and left only power: white which is the only one used when everything is booted properly, it was also necessary to set the default-on.

Reply:
Three comments later, but still I don't know which LEDs the device has and what they are supposed to do.

Reply:
The device has a single LED. Original behavior is described in the manual:
 
![image](https://user-images.githubusercontent.com/6545027/134810067-5a968b2c-8300-4ffe-b9c6-d0516df6a4ed.png)